docs/StardustDocs/topics/gettingStarted.md

```diff
@@ -1,22 +1,170 @@
-[//]: # (title: Getting started)
+[//]: # (title: Get started with DataFrame)
 
-The Kotlin DataFrame library is a JVM Kotlin library for in-memory data manipulation.
+Kotlin's DataFrame library gives you the power to manipulate data in your Kotlin projects. 
```

Comment:
We use "Kotlin DataFrame" across the documentation. It's the name of the library, sort of. At least it's what we agreed in the team. We can discuss it